1907-05-30.xml

```diff
@@ -1676,25 +1689,5586 @@
+                        present, the auditor is too muchaundet the heel of the directorate. This
+                        pointshould be carefully borne in mind by the members of’ the boards of
+                        directors of Anglo-Egyptian ‘companies for if it be neglected considerable
+                        trouble ‘will ensue.</p>
+                    <p>Apropos of Companies, a recent fine inflicted on an English company in Japan
+                        affords an interésting object lesson for Egypt. Besides Egypt and other
+                        parts of the Ottoman Empire, Capitulations exist in China, Japan and
+                        élsewhere. Japan, however, is stronger than Egypt and.is determined to
+                        prevent the scandal of limited liability companiés, whose field of
+                        operations is in Japan, being immtne from interference in this respect, His
+                        Majesty's Embassy at Tokio reports a ‘decision of some importance to British
+                        companies doing business in Japan, which was recenjly given in the case of a
+                        company whose head office is at Hong Kong. This company was prosecuted for
+                        not.complying »with certain regulations of Japanese Company Law, and was
+                        fined under Article 258 of the Commercial Code, which runs as follows. “A
+                        foreign company which sets up a principal office in Japan, or which makes it
+                        its principal object to do business in Japan, most, even though it is formed
+                        in a foreign country, comply with the same provisions as though it were
+                        formed in Japan.”</p>
+                    <p>It is a pity that the Egyptian Government cannot enact some such regulation,
+                        fore however wise may be the provisions of the English Companies acts, it is
+                        obvious to most people that we require here a good deal less latitude than
+                        is permitted at home.</p>
+                    <p>Presence of mind is the quality which above all others, the military\, man
+                        should cultivate. “Captain, we are entirely out of ammunition.” said an
+                        orderly sergeant to his commander on a field day. “What entirely out of
+                        ammunition ?” exclaimed the captain. “Yes, entirely out,” was the reply.
+                        “Then cease firing,” said the captain.</p>
+                </div>
             </div>
             <pb n="3"/>
-            <div type="page" n="3"> 
-                
+            <div type="page" n="3">
+                <div type="masthead">
+                    <head> THE BOYPTIAN GAZETTE, THURSDAY, MAY 30, 1907.</head>
+                </div>
+                <div type="article">
+                    <head>LOCAL AND GENERAL</head>
```

Comment:
Remember to change type="article" to section or items, just like what Dr. Hanley commented.